eng/Versions.props

```diff
@@ -9,7 +9,7 @@
         https://github.com/dotnet/arcade/blob/c788ffa83b088cafe9dbffc1cbc8155ba88b2553/Documentation/CorePackages/Versioning.md#output
     -->
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
-    <VersionPrefix>1.5.6</VersionPrefix>
+    <VersionPrefix>1.6.0</VersionPrefix>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <AssemblyVersion>1.0.0.0</AssemblyVersion>
```

Comment:
@eerhardt @ericstj the Assembly has been version 1.0.0.0 since we switched to arcade (and maybe before, can't remember). Should we also be bumping the assembly version to match the nuget package label of 1.6.0?

Reply:
We should be making our assembly version unique per release.  Can we do that at the start of the next release, so as not to introduce more changes into this release?

Reply:
Our versioning process should be to change versioning in main to the next milestone once we ship a release. Shipping a release should merely make the existing version stable (so as to change as little as possible in a release build)

Reply:
@ericstj thats normally what we do, but this time because we are adding in some major changes, like arm and a new version of IntelMKL, I thought we had decided to bump the version of this release to 1.6. If I understood incorrectly I am happy to move this back to 1.5.6 like it had before.

Reply:
No need to move it back.  I'm good with minor version bump.  Just making a comment for next time (primarily around the assembly versioning, so that we get feedback if anyone "notices" the assembly version bump).